deploy/rp-production.json

```diff
@@ -712,10 +712,17 @@
                     }
                 ],
                 "databaseAccountOfferType": "Standard",
-                "ipRangeFilter": "[if(parameters('disableCosmosDBFirewall'), '', concat('104.42.195.92,40.76.54.131,52.176.6.30,52.169.50.45,52.187.184.26', if(equals(parameters('extraCosmosDBIPs'), ''), '', ','), parameters('extraCosmosDBIPs')))]",
+                "ipRules": "[concat(if(parameters('disableCosmosDBFirewall'), createArray(), parameters('disableCosmosDBFirewall')), if(parameters('extraCosmosDBIPs'),parameters('extraCosmosDBIPs'), createArray())]",
```

Comment:
This is dependent on us changing the structure of the data that is presently a string pointer.  The type needs to be a []ipRangeOrAddress which we can do in the "configuration" to allow this to work.

Reply:
How do you think this changed in config repository? 
Currently we have "global hardcoded" IPs and per region ones. So overrides dont quite works. 
Can you give an example of how config will change?

Reply:
>How do you think this changed in config repository?

I don't believe they changed, but the newer version of the documentdb forces us to change the structure of the data in the config as we can not dynamically create an array of type `ipAddressOrRange` for each IP address where before they were string pointers being constructed as a string split by commas.

Currently, the config shows the IPs as "strings" but making them an array of type `ipAddressOrRange` should suffice.  
```yaml
extraCosmosDBIPs:
- ipAddressOrRange: some-ip-we-have
- ipAddressOrRange: some-ip-we-have
- ipAddressOrRange: some-ip-we-have
```

Reply:
Data is not generated but overridden in the `pkg/deploy/deploy_rp.go` where `mgmtdocumentdb.IPAddressOrRange` array is populated by the `extraCosmosDBIPs` from configuration as well as `disableCosmosDBFIrewall` parameters.